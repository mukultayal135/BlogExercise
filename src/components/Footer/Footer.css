footer {
  flex-grow: none;
  padding-bottom: 15px !important;
  background-color: black;
  color: white;
  font-size: large;
}
