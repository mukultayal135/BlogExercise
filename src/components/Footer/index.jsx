import React from 'react';
import './Footer.css';

function Footer() {
  return (
    <footer className="page-padding">
      <div>Â©artifact.com 2019</div>
    </footer>
  );
}

export default Footer;
