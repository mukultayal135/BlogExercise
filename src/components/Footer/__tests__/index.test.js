import React from 'react';
import { render } from '@testing-library/react';
import Footer from '..';

describe('Header', () => {
  it('should render correctly once more', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
