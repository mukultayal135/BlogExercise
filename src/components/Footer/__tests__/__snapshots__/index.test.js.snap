// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly once more 1`] = `
<DocumentFragment>
  <footer
    class="page-padding"
  >
    <div>
      Â©artifact.com 2019
    </div>
  </footer>
</DocumentFragment>
`;
